src/main/java/io/galeb/manager/handler/TargetHandler.java

```diff
@@ -120,6 +119,23 @@ public void beforeDelete(Target target) throws Exception {
     @HandleAfterDelete
     public void afterDelete(Target target) throws Exception {
         afterDelete(target, LOGGER);
+        waitSync(target);
```

Comment:
Exception not catched. TargetHandler.afterDelete must complete, even if an InterruptedException occurs